discojs/discojs-core/src/dataset/data/index.ts

```diff
@@ -2,4 +2,4 @@ export { DataSplit } from './data_split'
 export { Data } from './data'
 export { ImageData } from './image_data'
 export { TabularData } from './tabular_data'
-export { ImagePreprocessing } from './preprocessing'
+export { ImagePreprocessing, TabularPreprocessing } from './preprocessing'
```

Comment:
do we have tabular as a data modality in core disco now already? if yes can we re-base this PR?